src/middleware/getSkillsets.js

```diff
@@ -10,7 +10,7 @@ const getSkillSetList = store => next => async action => {
   const dispatch = store.dispatch;
 
   try {
-    const data = await fetch("api/skillSetList.json")
+    const data = await fetch("http://localhost:8087/candidates/skills")
```

Comment:
Please create a constant variable for host and please use it as follows, so that if we need to change the host in future we don't need to change every where.
export const constants = {
host:"localhost:8087"
}

import  {constants} from '../path/to/constans/'
 const data = await fetch(`${constants.host}/candidates/skills`)